terraform/deployments/govuk-publishing-platform/integration.backend

```diff
@@ -0,0 +1,4 @@
+bucket  = "govuk-ecs-terraform-integration"
```

Comment:
Should we be consistent on naming here with the `test` environment? So either:

`govuk-terraform-integration`

Or (for `test`)

`govuk-ecs-terraform-test`

Reply:
Yeah I picked this name to avoid any confusion with the old platform state files in integration, event though govuk-terraform-integration doesn't exist (we use govuk-terraform-state-integration and govuk-terraform-steppingstone-integration)
I feel it would be nice to have a clear delimitation now that the new platform will coexist with the current one